driver/kubernetes/driver.go

```diff
@@ -218,9 +225,19 @@ func (d *Driver) Client(ctx context.Context) (*client.Client, error) {
 	opts = append(opts, client.WithContextDialer(func(context.Context, string) (net.Conn, error) {
 		return conn, nil
 	}))
+
 	if td, ok := exp.(client.TracerDelegate); ok {
 		opts = append(opts, client.WithTracerDelegate(td))
 	}
+
+	if len(co.Meta) > 0 {
+		opts = append(opts, client.WithGRPCDialOption(grpc.WithChainUnaryInterceptor(func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
```

Comment:
I don't have all the context for this, but I think we probably want to add this to `grpc.WithChainStreamInterceptor` as well?

Reply:
As discussed with @tonistiigi, drop support for other drivers for now.